LibraryApp/src/app/components/CustomButton/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { Text, TouchableOpacity, ViewPropTypes } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+class CustomButton extends Component {
+  render() {
+    const { title, whiteTheme, buttonStyles, textStyles } = this.props;
+    return (
+      <TouchableOpacity style={[styles.button, whiteTheme && styles.whiteThemeButton, buttonStyles]}>
+        <Text style={[styles.text, whiteTheme && styles.whiteThemeText, textStyles]}>
+          {title.toUpperCase()}
```

Comment:
As I dont like having function calls inside the return, I rather creating a constant before return 
`const upperTitle = title.toUpperCase();
and then just 
`<Text ....>{upperText}....`